web-frontend/src/components/AccountListItem/AccountListItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { MenuActive } from '@massalabs/react-ui-kit';
+import AccountDefaultThumbnail from '../../assets/account-thumbnail-default.svg';
+import { Amount } from '@massalabs/react-ui-kit';
+
+export interface AccountListItemProps {
+  name: string;
+  amount: number;
+}
+
+export default function AccountListItem(props: AccountListItemProps) {
+  return (
+    <div className="flex flex-row justify-between align-center py-2 px-3 gap-48">
```

Comment:
This is too much! We must rethink that: `flex flex-row justify-between align-center py-2 px-3 gap-48`

It's just not maintainable, hard to follow and understand. So we are adding something to make our life hard.

Reply:
what do you suggest to simplify this? 

I found this way: https://stackoverflow.com/a/73899407/5903242 but we must keep in mind that we want to share those aliases from the ui-kit